contents/copy/scholarships/guide.md

```diff
@@ -1,3 +1,114 @@
 ---
-title: Scholarships Guide
+title: Scholarship Holder's Guide
+introText: All of the necessary information you need to know to make your experience at CSSconf EU 2018 a blast!
 ---
+
+## Welcome!
+
+Thank you so much for joining us for CSSconf EU and/or JSConf EU 2018! We're really excited to see you soon in Berlin, but until then, we've created this page to help you with any roadblocks you may be facing on your way there.
+
+## Hotel Information
+
+### [A&O Berlin Friedrichshain](https://www.aohostels.com/en/berlin/berlin-friedrichshain/)
+
+*Boxhagener Str. 73, 10245 Berlin*
+
+We have booked a room for you from:
+- Thursday to Monday for combo ticket holders
+- Thursday to Saturday for CSSconf ticket holders
+- Friday to Monday for JSConf ticket holders
+
+(that is, unless you have private accommodation). In case you need assistance with regards to the hotel, you can contact Simone (simone@jsconf.eu).
+
+If you're arriving by public transport, the closest stations are:
+
+- **Berlin Ostkreuz** (S-Bahn, Bus) — One of the biggest train stations in Berlin, it's serviced by the S3, S5, S7, Ring (S41, S42), and more S-Bahn lines.
+- **Neue Bahnhofsstr.** (Tram, Bus) — Situated right next to the hotel and serviced by the 21 tram and the 240 bus line.
+
+You will only need an *AB Ticket* to get there. For more information, refer to the transport section.
+
+## Venue Information
+
+### Arena Berlin
+
+*Eichenstraße 4, 12435 Berlin*
+
+The entire event, including meals, parties, and everything else takes place in here, so there's no long distances you have to take during the day.
+
+If you're travelling to the venue by taxi, most taxi drivers should recognize the destination if you just say "Arena Berlin". The closest public transport stations are:
+
+- **Treptower Park** (S-Bahn, Bus) — Serviced by the S8, S9, and the Ring lines (S41 and S42). From there, you can walk or take the 165/265 bus lines.
+- **Eichenstraße/Puschkinallee** (Bus) — Serviced by exactly those 165/265 lines, as well as the 104 line. From here, the venue is a stone throw away. This station is also reachable from Schlesisches Tor.
+
+## Bookings & Reimbursements
+
+If you agreed to it, we have booked the hotel for you. In case you're unable to affort transportation costs, such as taxis or public transport, please contact Simone and we'll figure something out together!
+
+## Contacts
+
+If you have any questions or need help, feel free to talk to us! We're all on the "wwwtf.berlin" Slack instance that you've been invited to.
+
+- **Simone Haas,** simone@jsconf.eu, `@haas_simone` on Twitter
+- **Olivia Hugger**, `@serbal_chan` on Twitter
+- 
```

Comment:
Outside of Simone, Princiya (who still needs to be added here) and myself, should there be any further contact person?

Reply:
Felicitas Kugland, feli@jsconf.eu, @kotzendekrabbe is the person in charge for travel booking. Those who have travel grants and questions concerning their bookings should get in contact with her